app/platform/server_license.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package platform
+
+import (
+	"github.com/mattermost/mattermost-server/v6/model"
+)
+
+func (ps *PlatformService) License() *model.License {
+	license, _ := ps.licenseValue.Load().(*model.License)
+	return license
+}
```

Comment:
Hmm.. I see the same method in the `*Server` struct as well. Any reason we need to have both?

Reply:
This was also a temp step, will/should be fixed via https://mattermost.atlassian.net/browse/MM-45839 I'm going to add a comment nonetheless.

Reply:
Yep, let's mention the ticket number in the comment. Thanks!